actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -77,6 +79,9 @@
     @Wired
     Container container;
 
+    @Inject
```

Comment:
Yes, that's correct. In other cases where that might happen, we have the @Config and we provide manual getter/setter for those not running in container. You can see that with the Configs in stage, and how we set up the samples which aren't running container (Hello).

With @Inject it will just be null not running in container, so we should probably provide a static getting or similar and use that. I assume dropwizard is thread safe, so we'll just need to be sure our wrapper is.